sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeComposeScreen.kt

```diff
@@ -0,0 +1,30 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import co.nimblehq.sample.compose.model.UiModel
+import co.nimblehq.sample.compose.ui.screens.AppBar
+
+@Composable
+fun HomeComposeScreen(
```

Comment:
I think we can naming as `HomeScreen` same as `HomeFragment` and `HomeViewModel`

Reply:
Seem like @Tuubz don't agree on this 😄 https://github.com/nimblehq/android-templates/pull/272#discussion_r947660564

Reply:
Conclusion: I will leave TODO comments for now and we will consider renaming them when we [completely opt-out Fragment structure](https://github.com/nimblehq/android-templates/discussions/279#discussioncomment-3412071)